cns/NetworkContainerContract.go

```diff
@@ -18,6 +18,7 @@ import (
 const (
 	SetOrchestratorType                      = "/network/setorchestratortype"
 	GetHomeAz                                = "/homeaz"
+	GetVMUniqueID                            = "/vmuniqueid"
```

Comment:
too late I suppose, but I wish we would have put the routes like home az under a `/metadata` path, then vm unique ID could have gone under there too

Reply:
I still made it as /metadata/vmuniqueid. At least any future metadata APIs can adhere to it.